author/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
```

Comment:
this class should inherit the items it is supposed to be like

` class Author < items`